conn/physic/example_test.go

```diff
@@ -423,8 +423,10 @@ func ExampleLuminousIntensity_flag() {
 
 func ExampleEnergy() {
 	fmt.Println(1 * physic.Joule)
+	fmt.Println(1 * physic.KiloWattHour)
```

Comment:
Probably should add examples for the new constants, as they are great for increasing visibility.

Reply:
Added one for WattSecond.